defaults/test_default.yaml

```diff
@@ -312,3 +312,5 @@ mgmt_snapshots_preparer_params:
   num_of_rows: ''
   sequence_start: ''
   sequence_end: ''
+
+adaptive_timeout_store_metrics: true
```

Comment:
let's make it `false` for docker backend (assuming we drop node_exporter on it after merging https://github.com/scylladb/scylla-cluster-tests/pull/10627)